web/src/main/webapp/WEB-INF/classes/setup/sql/data/data-db-default.sql

```diff
@@ -731,6 +731,7 @@ INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('system
 INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('system/security/passwordEnforcement/usePattern', 'true', 2, 12002, 'n');
 INSERT INTO Settings (name, value, datatype, position, internal, editable) VALUES ('system/security/passwordEnforcement/pattern', '^((?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*(_|[^\w])).*)$', 0, 12003, 'n', 'n');
 
+INSERT INTO Settings (name, value, datatype, editable, encrypted, position, internal) VALUES ('metadata/batchediting/accesslevel', '', 0, 'y', 'n', 180001, 'n');
```

Comment:
I believe this logic is similar to metadata/import/userprofile so I would suggest the following:
`
INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('metadata/import/userprofile', 'Editor', 0, 12001, 'n');`

- Move this line change under line 694
- remove columns editable and encrypted because defaults should be ok
- Change the position to 12020 because 180001 seems to be too high.
- Set the default to Editor as that is what was used prior to the new feature.

Reply:
resolved